packages/perseus/src/components/lint.tsx

```diff
@@ -64,43 +61,12 @@ type State = {
  * that has a right margin (like anything blockquoted) the circle will appear
  * to the left of where it belongs.  And if there is more
  **/
-class Lint extends React.Component<Props, State> {
+class Lint extends React.Component<Props> {
```

Comment:
I love that we can improve/simplify the tooltip code for both the existing iframe and the new preview component.